cl/citations/tasks.py

```diff
@@ -214,7 +206,7 @@ def store_opinion_citations_and_update_parentheticals(
                 )
 
     # Finally, commit these changes to the database in a single
-    # transcation block. Trigger a single Solr update as well, if
+    # transcation block. Trigger a single update as well, if
     # required.
```

Comment:
This whole step is removed, so we can remove the comment about it.

```suggestion
    # transcation block. 
```

Reply:
alright!